chartlets.py/chartlets/components/typography.py

```diff
@@ -5,4 +5,14 @@
 
 @dataclass(frozen=True)
 class Typography(Component):
+    """Use typography to present your design and content as clearly
+    and efficiently as possible."""
+
     text: str | None = None
+    """Optional text to be displayed."""
+
+    color: str | None = None
+    """The color of the component."""
+
+    variant: str | None = None
+    """Applies the theme typography styles."""
```

Comment:
Did you mean `Applies the theme to typography styles` or `Applies the theme/typography styles`?

Reply:
Copied from https://mui.com/material-ui/api/typography/